src/Features/Core/Portable/InlineParameterNameHints/IInlineParamNameHintsService.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.InlineParameterNameHints
+{
+    readonly struct NameAndSpan
```

Comment:
Maybe call this InlineParameterHint or something so it's a bit clearer what it is?